src/components/SettingsAdvanced.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+    <div class="kiwi-settings-advanced">
+        <div class="kiwi-settings-advanced-notice">{{ $t('settings_advanced_header') }}</div>
+        <form class="u-form">
+            <div class="kiwi-settings-advanced-filter">
+                <div class="kiwi-settings-filter-container">
+                    <input id="filter" v-model="filterString"
+                           :placeholder="$t('settings_advanced_filter')"
+                           class="u-input">
+                    <i v-if="!filterString" class="fa fa-search" aria-hidden="true"/>
+                    <i v-else class="fa fa-times" aria-hidden="true" @click="filterString = ''"/>
+                </div>
+            </div>
+            <table class="u-table kiwi-settings-advanced-table" cellspacing="0">
+                <p v-if="filteredSettings.length === 0" class="kiwi-settings-advanced-noresult">
+                    {{ $t('settings_advanced_noresult') }}
+                </p>
+                <tr v-for="setting in filteredSettings" v-else
+                    :key="setting.key"
+                    :style="{'font-weight': (setting.modified) ? 'bold' : 'normal' }">
+                    <td><label :for="setting.key">{{ setting.key }}</label></td>
+                    <td v-if="setting.modified">
+                        <a class="u-link" @click="resetValue($event, setting.key)">
+                            {{ $t('settings_advanced_reset') }}
+                            <i class="fa fa-undo reset-icon" style="margin-left: 10px;"/>
+                        </a>
+                    </td>
+                    <td v-else />
+                    <td>
+                        <input v-if="setting.type === 'boolean'"
+                               :checked="setting.val"
+                               :id="setting.key"
+                               type="checkbox"
+                               @click="updateSetting($event, setting.key)">
+                        <input v-else-if="setting.type === 'number'"
+                               :value="setting.val"
+                               :id="setting.key"
+                               class="u-input"
+                               type="number"
+                               @keydown.13="$event.target.blur()"
+                               @change="updateSetting($event, setting.key)"
+                               @blur="updateSetting($event, setting.key)">
+                        <input v-else
+                               :value="setting.val"
+                               :id="setting.key"
+                               class="u-input"
+                               @keydown.13="$event.target.blur()"
+                               @blur="updateSetting($event, setting.key)">
```

Comment:
All of these IDs need prefixing with something to prevent clashes with anything else on the page. Eg. a setting called `filter` would then clash with an element elsewhere on the page with the ID `filter`.